lib/components/FileAttachment/index.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useRef } from "react";
+
+import { saveAs } from "file-saver";
+
+import directUploadsApi from "apis/direct_uploads";
+import { DIRECT_UPLOAD_ENDPOINT } from "common/constants";
+import Button from "components/Common/Button";
+import useUppyUploader from "hooks/useUppyUploader";
+
+import Attachment from "./Attachment";
+import {
+  DEFAULT_UPPY_CONFIG,
+  DROP_DOWN_OPTIONS_KEYS,
+  UPPY_UPLOAD_CONFIG,
+} from "./constant";
+
+import Progress from "../Editor/MediaUploader/Progress";
+
+const FileAttachment = ({
+  endpoint = DIRECT_UPLOAD_ENDPOINT,
+  attachments = [],
+  onChange = _ => {},
+}) => {
+  const fileInputRef = useRef(null);
+  const uppyConfig = { ...DEFAULT_UPPY_CONFIG };
+
+  const { uppy, isUploading } = useUppyUploader({
+    endpoint,
+    uppyConfig,
+    uppyUploadConfig: { ...UPPY_UPLOAD_CONFIG },
```

Comment:
Same comment as above. Why the unnecessary destructure? The `uppyUploadConfig` will not be modified in anyway by uppy.io.